components/DetailsPageHighlights/detailspagehighlights.scss

```diff
@@ -0,0 +1,183 @@
+.bx--artifact-details-highlight-container {
+
+  & .bx--createArtifactFlexGroupColumn {
+    margin-bottom: rem(35px);
+  }
+
+  & .bx--createArtifactFlexGroupColumn {
+    display: flex;
+    flex-direction: column;
+
+    &.bx--hide {
+      display: none;
+    }
+  }
+
+  & .bx--title-label {
+    font-size: 1.55rem;
+    font-weight: 200;
+    line-height: 1.5;
+  }
+
+  & .bx--description-label {
+    font-size: rem(14px);
+    font-weight: 300;
+    margin: rem(25px) 0;
+  }
+
+  @include breakpoint(825px) {
+
+    & .bx--createArtifactFlexGroupRow {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      flex-wrap: wrap;
+    }
+  }
+}
+
+.bx--artifact-details-description-container {
+  margin-bottom: 5px;
+}
+
+@media (max-width: 768px) {
+  .bx--artifact-details-description-container {
+    width: calc(100% - 100px);
+  }
+}
+
+.bx--description-label,
+.bx--markdown-container h1 {
+    margin: 0 0 15px 0;
+    padding: 0;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    font-size: 1.55rem;
+    font-weight: 300;
+    line-height: 1.5;
+    letter-spacing: normal;
+}
+
+.bx--artifact-details-highlight-container {
+  .bx--markdown-container {
+    .underline-text {
+      text-decoration: underline;
+    }
+
+    em {
+      font-style: italic;
+    }
+
+    p {
+      margin-bottom: .625rem;
+
+      &:last-child {
+        margin-bottom: 0;
+      }
+    }
+
+    strong {
+      font-weight: bold;
+    }
+
+    sup {
+      vertical-align: super;
+      font-size: smaller;
+    }
+
+    ul {
+      padding: 1rem;
+      list-style-type: disc;
+
+      ul {
+        list-style-type: none;
+
+        li::before {
+          content: "-";
+          padding-right: .3125rem;
+        }
+
+        li:last-child {
+          margin: 0;
+        }
+      }
+
+      > li {
+        margin-bottom: 1.5rem;
+        padding-left: .25rem;
+      }
+    }
+  }
+}
+
+
+.bx--artifact-details-description,
+.bx--markdown-container {
+  font-size: 0.875em;
+  font-family: "IBM Helvetica", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+}
+
+.bx--artifact-details-description-content {
+  width: 100%;
+  height: auto;
+  overflow: hidden;
+  line-height: 1.2858;
+
+  .bx--artifact-details-description-bullets {
+    display: flex;
+    justify-content: flex-start;
```

Comment:
flex-start is the initial starting property for `justify-content`, no need to describe it here